src/editors/playgroundController.ts

```diff
@@ -286,7 +289,10 @@ export default class PlaygroundController {
         .then(undefined, (error) => {
           log.error('Evaluate playground with cancel modal error', error);
 
-          return resolve(undefined);
```

Comment:
typescript & lint wasn't happy with this line which makes sense. This is an unrelated change.
I'll make a ticket to look into adding lint checks to this project.